config/dune

```diff
@@ -22,4 +22,4 @@
    plugin_list
   (env_var PWD) ; used in the fallback of COQ_CONFIGURE_PREFIX
   (env_var COQ_CONFIGURE_PREFIX))
- (action (chdir %{project_root} (run %{project_root}/tools/configure/configure.exe -no-ask -native-compiler no))))
```

Comment:
I'd prefer this to remain explicit.

Reply:
I thought about it and wondered why this should be the only explicit argument here, so this is why I removed it. But happy to revert.

Reply:
As you want, in my case it is just a preference. For the rest, modulo `-prefix` (which is dealt by -no-ask, but maybe should not), I think that we rarely change the options in real use.